### ISSUE TITLE
Display Clarity Improvements

### ISSUE THREAD

**Issue by peterhuson** [peterhuson/fsae-apollo#3]
I've found that the default screen we currently display is not super easy to read while driving. It would be a good idea to modify the font, size, and alignment to clarify the values that are shown. 

### Rounding
Most values should display no decimal values, or a single decimal point to reduce clutter. I would vote for leaving Vbat, and lambda with one decimal point. 

### Font and size
We should really switch to a block font rather than the current serif we use. Also, we should make the font massive and only leave space in between numbers for borders. 

### Text Alignment
One important thing is to fix the alignment of the values such that the addition of another digit to a number doesn't shift the number and confuse the driver. (i.e. when RPM goes from 9999 to 10000 the 1 should appear to the left of where the 9999 was without shifting the position of the entire number). See below:
![Capture2](https://user-images.githubusercontent.com/9157695/66166422-49451580-e605-11e9-8b11-c2268f3feded.PNG)
![Capture](https://user-images.githubusercontent.com/9157695/66166423-49451580-e605-11e9-954c-b496f410a323.PNG)

**Comment by MLavrentyev** [comment]
Might also make sense to look at a different font - Eurostile seems to be the go-to for dashboards since it's blocky enough to be easy to read (and the numbers are monospace making number alignment easy).

https://fonts.adobe.com/fonts/eurostile

**Comment by peterhuson** [comment]
I agree, the single font I’ve found so far that works in Qt on the NanoPi is Courier. It works ok but I would definitely want to figure out how to use a better font of our choosing.
